docs/fundamentals/code-analysis/style-rules/ide0130.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "IDE0130: Namespace does not match folder structure"
+description: "Learn about code analysis rule IDE0130: Namespace does not match folder structure"
+ms.date: 12/05/2021
+ms.topic: reference
+f1_keywords:
+- IDE0130
+helpviewer_keywords:
+- IDE0130
+author: gewarren
+ms.author: gewarren
+dev_langs:
+- CSharp
+- VB
+---
+# Namespace does not match folder structure (IDE0130)
+
+| Property                 | Value                                                  |
+| ------------------------ | ------------------------------------------------------ |
+| **Rule ID**              | IDE0130                                                |
+| **Title**                | Namespace does not match folder structure              |
+| **Category**             | Style                                                  |
+| **Subcategory**          | Naming  rules                                          |
```

Comment:
```suggestion
| **Subcategory**          | Language rules                                        |
```

Reply:
I don't think this is a "naming rule" in terms of how "naming rules" are used in the docs.

Reply:
Agree. I had this change and forgot to hit save :/